config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -3710,10 +3714,14 @@
 # proctorio.options=recordvideo, recordaudio, recordscreen, recordwebtraffic, recordroomstart, verifyvideo, verifyaudio, verifydesktop, verifyidauto, verifyidlive, verifysignature,
 #     fullscreenlenient, fullscreenmoderate, fullscreensevere, clipboard, notabs, linksonly, closetabs, onescreen, print, downloads, cache, rightclick, noreentry, agentreentry, calculatorbasic, calculatorsci, whiteboard
 
-# SAK-46501 property to select the default sorting column in assessment list view
-# The column by which we want to sort the table
-# DEFAULT: 2
-# samigo.assessmentSortingColumn=9
+# Safe exam Browser can be enabled globally or on a per-site basis
```

Comment:
Please put the S2U-21 code before 'Safe Exam Browser'.
And put a link to the SEB website:

Ability to conduct safe exams using the 'Safe Exam Browser' open source tool
More information about SEB https://safeexambrowser.org/about_overview_en.html